pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -541,7 +549,11 @@ func (ctrl *Controller) syncContainerRuntimeConfig(key string) error {
 				tempIgnCfg := ctrlcommon.NewIgnConfig()
 				mc = mtmpl.MachineConfigFromIgnConfig(role, managedKey, &tempIgnCfg)
 			}
-			mc.Spec.Config = createNewCtrRuntimeConfigIgnition(storageTOML, crioTOML)
+			tomlConfig := map[string][]byte{
```

Comment:
(Absolutely non-blocking: inlining the map constructor into the call would avoid a single-use short-term variable. This works perfectly fine as well, feel free to keep it this way if it is easier to read for you. [Both callers of `createNewIgnition` should be consistent in this respect. See the comments in `createNewIgnition` first, though.])

Reply:
Fixed